config/initializers/wrap_parameters.rb

```diff
@@ -5,7 +5,7 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
+  wrap_parameters format: false
```

Comment:
What is this for? 🤔

Reply:
https://github.com/heartcombo/devise/issues/4825

Reply:
Thanks for the context! However Isn't this necessary for other endpoints to work as expected? If you confirm it is not required for other endpoints, then I'm fine with it, but we should double check it doesn't affect current behavior just in case

Reply:
It doesn't look like it is required for other endpoints, every endpoint is working as intended even with this change implemented

Reply:
Nvm, it does mess with the existing endpoints. Will remove.